src/istio/control/tcp/mock_check_data.h

```diff
@@ -28,6 +28,8 @@ class MockCheckData : public CheckData {
  public:
   MOCK_CONST_METHOD2(GetSourceIpPort, bool(std::string* ip, int* port));
   MOCK_CONST_METHOD2(GetPrincipal, bool(bool peer, std::string* user));
+  MOCK_CONST_METHOD2(GetSourceNamespace,
+                     bool(const std::string&, std::string* ns));
```

Comment:
did we have this virtual function in the base class?

Reply:
No, I forgot to remove this.